src/client/AppRouter.tsx

```diff
@@ -22,8 +23,9 @@ export const AppRouter: React.FunctionComponent<{}> = () => {
                     component={ClassroomPageContainer}
                 />
                 <Route path="/room/:roomId" component={PrivateRoomContainer} />
-                <Route path="/login" component={LoginContainer} />
-                <Route path="/register" component={RegisterContainer} />
+                <Route path="/calendar-test" component={Calendar} />
+                ] <Route path="/login" component={LoginContainer} />
```

Comment:
Brackets?

Reply:
The brackets may be accidentally left there when I was resolving merge conflicts

Reply:
Bracket removed